app/controllers/purchase_job_postings_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PurchaseJobPostingsController < ApplicationController
```

Comment:
I would name this with a noun given that it's a resource. Something like: `JobPostingPurchases`

```ruby
PurchaseJobPostings.create # less natural
JobPostingPurchases.create # more natural
```